content/v2.0/visualize-data/other-tools/grafana.md

```diff
@@ -1,72 +1,114 @@
 ---
-title: Use Grafana with InfluxDB 2.0
+title: Use Grafana with InfluxDB
 description: >
-  Use the Grafana Flux datasource plugin to connect Grafana to InfluxDB 2.0 or
-  InfluxDB Cloud 2.0.
+  Use [Grafana](https://grafana.com/) to visualize data from **InfluxDB 2.0** and **InfluxDB Cloud**.
 menu:
   v2_0:
     name: Use Grafana
     parent: Other visualization tools
 weight: 201
 v2.0/tags: [grafana]
+related:
+  - https://grafana.com/docs/, Grafana documentation
```

Comment:
is this 'grafana documentation' supposed to be in here after the URL?

Reply:
Yeah, under `related`, you can add external related links. Everything before the comma is the link URL. Everthing after the comma is the link text.